Lib/fontmake/font_project.py

```diff
@@ -236,7 +238,9 @@ def _font_name(self, ufo):
     def _output_dir(self, ext, is_instance=False):
         """Generate an output directory."""
 
-        dir_prefix = 'instance_' if is_instance else 'master_'
+        dir_prefix = os.path.join(
+            'out_compatible' if self.compatible else 'out',
```

Comment:
That could also work, though my thinking was that twelve base output directories `(master/instance)_(ufo/otf/ttf)[_interpolatable]` gets a bit messy, so better to put everything in either an out/ or out_interpolatable/ directory. But it's subjective.